src/docs/sdk/performance/dynamic-sampling-context.mdx

```diff
@@ -99,15 +99,16 @@ SDKs should recognize incoming requests as "instrumented" or "traced" when at le
 - The incoming request has a `sentry-trace` header
 - The incoming request has a `baggage` header containing one or more keys starting with "`sentry-`"
 
-After the DSC of a particular trace has been frozen, API calls like `set_user` or `set_transaction` should have no effect on the DSC.
+After the DSC of a particular trace has been frozen, API calls like `set_user` should have no effect on the DSC.
 
 ## Payloads
 
 Dynamic Sampling Context is sent to Sentry via the `trace` envelope header and is propagated to downstream SDKs via a baggage header.
 
 All of the values in the payloads below are required (non-optional) in a sense, that when they are known to an SDK at the time a transaction envelope is sent to Sentry, or at the time a baggage header is propagated, they must also be included in said envelope or baggage.
 
-### Handling Sensitive Data and PII
+<!-- Comment on why the following section is commented out: Without `transaction` and `user_id` in the DSC there aren't any PII concerns, we might want to use them in the future though.  -->
```

Comment:
We have Git history for bringing this section back. So why not delete the section for now?

Reply:
Yea true. Thanks! --> [b714ed2](https://github.com/getsentry/develop/pull/631/commits/b714ed28b03d10011340defa665c9ab7526e129e)